### ISSUE TITLE
Exception while using <Center /> component

### ISSUE THREAD

**Issue by mir3z** [gakimball/react-inky#14]
Consider the following component:
```jsx
const Foo = ({ children }) => (
    <Center>
        { children }
    </Center>
)
```

Then using it like that:
```jsx
<Foo>Hello!</Foo>
```

In this case `<Center />` will fail to render with `TypeError: Cannot read property 'className' of undefined` exception.

**Comment by gakimball** [comment]
What's the value of `children`?

**Comment by gakimball** [comment]
Published a fix in `0.3.1`

**Comment by mir3z** [comment]
Thanks for quick response.
